dependencies/src/main/java/com/google/cloud/tools/opensource/dependencies/DependencyGraphBuilder.java

```diff
@@ -339,17 +339,15 @@ private DependencyGraphResult levelOrder(
             for (ArtifactResult artifactResult :
                 resolutionException.getResult().getArtifactResults()) {
               if (artifactResult.getArtifact() == null) {
-                DependencyNode failedDependencyNode = artifactResult.getRequest().getDependencyNode();
-                ExceptionAndPath failure =
-                    ExceptionAndPath.create(parentNodes, failedDependencyNode, resolutionException);
-                artifactProblems.add(new UnresolvableArtifactProblem(failure.getPath()));
+                DependencyNode failedDependencyNode =
+                    artifactResult.getRequest().getDependencyNode();
+                artifactProblems.add(
+                    new UnresolvableArtifactProblem(
+                        failedDependencyNode.getArtifact(), parentNodes));
```

Comment:
```suggestion
                        ImmutableList.builder().addAll(parentNodes).add(failedDependencyNode).build()));
```

In `ClassPathBuilderTest.testResolve_artifactProblems` failure, the last element of parentNodes does not match `failedDependencyNode.getArtifact()`, because "ant:1.6.2" is not the missing artifact.

![image](https://user-images.githubusercontent.com/28604/74058917-03ce9580-49b5-11ea-80d4-95339962b6a7.png)